flipperdevices/flipperzero-firmware
# Lfrfid direct emulation

# What's new

- Created some new scenes in applications/lfrfid/scenes
- lfrfid_app_scene_emu_data
- lfrfid_app_scene_emu_menu
- Also updated some core files to enumerate the new scenes

# Verification 

- Recompile the firmware and load it on ur flippy
- Check "125 kHz RFID" application for a 4th menu item - Emulate Directly
- Select encoding
- Enter hex digits and hit "save"
- Flippy should now be emulating the digits u entered
- Hitting back will take u back to byte edit screen

# Checklist (For Reviewer)
